lib/Specstrom/Evaluator.hs

```diff
@@ -109,7 +116,11 @@ initialEnv =
       ("always_", PartialOp AlwaysOp []),
       ("eventually_", PartialOp EventuallyOp []),
       ("next_", PartialOp NextOp []),
-      ("access", PartialOp MkAccessor [])
+      ("access", PartialOp MkAccessor []),
+      ("text", Independent $ LitVal (ElementStateLit Text)),
+      ("cssValue", PartialOp MkCssValue []),
+      ("attribute", PartialOp MkAttribute []),
+      ("property", PartialOp MkProperty [])
```

Comment:
If these are literals, which they appear to be, they shouldn't be in the list of built-in ops. You should generate them directly from the parser instead.

Reply:
Although it would make quickLTL more Web-specific to bake it in the parser, I don't think that matters for now

Reply:
Hmm, yeah maybe that's better to ensure they're literals (and thus statically known). I'll change it, might need some help to figure out how.